tablist.el

```diff
@@ -309,7 +309,7 @@ as argument for the function `completion-in-region'.")
 
 (defun tablist-init (&optional disable)
   (let ((cleaned-misc (cl-remove 'tablist-current-filter
-                                 mode-line-misc-info :key 'car)))
+                                 mode-line-misc-info :key 'car-safe)))
```

Comment:
Looks good.  While you're at: Could you replace this with proper function-quoting (`#'car-safe`) ? Thanks.